Apps/Common/src/AccessManagement/Authorization/Policy/MedicationPolicy.cs

```diff
@@ -21,13 +21,23 @@ namespace HealthGateway.Common.AccessManagement.Authorization.Policy
     public static class MedicationPolicy
     {
         /// <summary>
-        /// Policy which allows the reading of the identified patient.
+        /// Policy which allows the reading medication statements.
```

Comment:
/// Policy which allows the reading _of_ medication statements.

Reply:
fixed